.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    webextensions: true
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:mozilla/recommended"
+  ],
+  globals: {
+    addMessageListener: false,
+    sendAsyncMessage: false,
+    content: false
```

Comment:
I couldnt figure out where `addMessageListener()` lives, or any supported ESLint environment that defines those as globals, so I resorted to just manually specifying them.